src/bitfield/mod.rs

```diff
@@ -163,11 +226,11 @@ impl Bitfield {
         let right = get_index_value(value) >> tree_index(o);
         let mut byte = left | right;
         let len = self.index.len();
-        let max_len = self.page_len * 256;
+        let max_len = self.data.page_len() * 256;
 
         self.iterator.seek(start);
 
-        while self.iterator.index() < max_len
+        while self.iterator.index() < max_len.try_into().unwrap()
```

Comment:
Could this be fixed with a `as _` instead of unwraping? My concern is to include panics on the code here

Reply:
addressed